src/views/Dashboard/Customer/Home/RequestTest/PaymentDetailCard.vue

```diff
@@ -344,7 +344,7 @@ export default {
         return
       }
 
-      if (this.remainingStaking && this.remainingStaking > 0) {
+      if (this?.remainingAmount > 0) {
```

Comment:
I think this is not the best way, please use the `&&` operator instead optional chain

Reply:
done mas @bukandicki . please kindly re-review. thanks :)